content/md/assets/upload-assets.md

```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
 
 Mass upload assets is pretty convenient especially if you want to update your asset catalog with external sources: photo shooting, new collection, etc...
 
-The PIM uses the filename to know for which asset the file have to be used.
+The PIM uses the filename to know for which asset the file have to be used and if this asset is localized or not.
 
-**Example 1**  
+**Example 1 with a not localized asset**  
 If the filename is `main_picture_S1263547.jpg`, then the PIM will check:
-- If the asset with the `main_picture_S1263547` code already exists, the PIM will update the asset with the `main_picture_S1263547` code by importing the `main_picture_S1263547.jpg` file as its reference file (and generate the variation files accordingly)
-- If the asset doesn’t already exist, the PIM will create a new asset with the `main_picture_S1263547` code and with the file as the reference file, so that the variation files can be generated
+- If the asset with the `main_picture_S1263547` code already exists, the PIM will update the asset with the `main_picture_S1263547` code by importing the `main_picture_S1263547.jpg` file as its reference file (and generate the variation files accordingly).
```

Comment:
as its new reference file (and will regenerate the variation files...)

Reply:
done